.github/workflows/yarn_docker_image.yml

```diff
@@ -11,6 +11,10 @@ on:
                 default: "18"
                 type: string
                 required: false
+            yarn_version:
+                default: "1.22.19"
```

Comment:
Default to latest? Or LTS?

Reply:
Cannot set latest, see the usage of the variable in the curl.